regression/tests/enterprise/test_enterprise_data_consent.py

```diff
@@ -0,0 +1,126 @@
+    CONSENT_DECLINE_WARNING = "We could not enroll you"
+
+    def setUp(self):
+        """
+        Initialize all page objects
+        """
+        super(TestEnterpriseDataConsent, self).setUp()
+        self.browser.maximize_window()
+        self.enterprise_data_consent = EnterpriseDataConsentPage(self.browser)
+
+    def test_data_consent_page_details(self):
+        """
+        Scenario: To verify that user sees the correct settings on data consent
+            page
+            Given that a new user registers for the course through enterprise
+            When this user clicks next from enrollment page
+            Then this user is taken to data consent page
+            And relevant details are shown on this page
+        """
+        # The edX site is visited just to make sure that when user jumps to
+        # edX from portal we don't have to handle authentication popup
+        self.lms_login.visit()
+        # Enterprise portal flow
+        self.login_to_enterprise_portal()
+        self.access_course()
+        self.register_enterprise_edx_user()
+        # Verify that user is on course enrollment page and correct course
+        # is displayed there
+        self.enterprise_course_enrollment.wait_for_page()
+        self.assertDictEqual(
+            self.ENT_COURSE_TITLE,
+            self.enterprise_course_enrollment.get_course_title()
+        )
+        self.enterprise_course_enrollment.continue_to_data_consent_page()
+        self.enterprise_data_consent.wait_for_page()
+        # Verify consent text
+        self.assertIn(
+            self.CONSENT_MSG,
+            self.enterprise_data_consent.get_consent_message_text()
+        )
+        # Verify org name in consent text
+        self.assertEqual(
+            self.ENT_COURSE_ORG,
+            self.enterprise_data_consent.get_enterprise_name_from_msg()
+        )
+        # Verify that user is able to open policy dropdown from policy link
+        self.enterprise_data_consent.open_policy_text()
+
+    def test_data_consent_acceptance(self):
+        """
+        Scenario: To verify that user is taken to basket page on accepting
+            data consent
+            Given that a user is on data consent page
+            When this user checks the data consent check box
+            And Clicks on the Continue button
+            Then this user is taken to basket page
+        """
+        # The edX site is visited just to make sure that when user jumps to
+        # edX from portal we don't have to handle authentication popup
+        self.lms_login.visit()
+        # Enterprise portal flow
+        self.login_to_enterprise_portal()
+        self.access_course()
+        self.register_enterprise_edx_user()
+        # Verify that user is on course enrollment page and correct course
+        # is displayed there
+        self.enterprise_course_enrollment.wait_for_page()
+        self.assertDictEqual(
+            self.ENT_COURSE_TITLE,
```

Comment:
Please use util method `go_to_enterprise_course_enrollment_page` to get to enterprise course enrollment page as suggested above.